DfE.FindInformationAcademiesTrusts/Pages/Error.cshtml

```diff
@@ -33,7 +33,7 @@
           <a class="govuk-link" href="@ViewConstants.ReportAProblemMailToLink">Report this problem</a>
         </feature>
         <feature name="@FeatureFlags.UpdatedFooterHelpLink">
-          <a class="govuk-link" href="@ViewConstants.GetHelpFormLink">Report this problem</a>
+          <a class="govuk-link" href="@ViewConstants.GetHelpFormLink" rel="noopener" target="_blank">Report this problem</a>
```

Comment:
For a11y, we should indicate to the user when something opens in a new tab. In some places we're adding "(opens in a new tab)" in the link text, in some (like "More information" on the academies details table) we're putting the extra info in aria.